README.rst

```diff
@@ -559,9 +559,10 @@ These files are expected to be Python scripts defining two variables:
         a fpm_ command-line option. If the key is only one character (for
         example ``c``), it will be passed as ``-<key><value>`` and if it's
         more, it will be passed as ``--<key>=<value>`` (``_`` characters in the
-        key will be replaced by ``-`` for convenience). No validation is
-        performed over the keys or values, they are just passed blindly to
-        fpm_.
+        key will be replaced by ``-`` for convenience). The ``<value>`` can be a
+        string or an array of strings. In the latter case, the key is used as fpm 
```

Comment:
`fpm_` so it get a pretty link. Also maybe `used as the fpm_ flag` (I would drop the "cli" too, as it's the only interface fpm offers).